### DISCUSSION TITLE
How can i debug the hax after injecting to game? or such like 'hot reload'.

### DISCUSSION THREAD

**Discussion by GazzyLifeSense** [winstxnhdw/lc-hax#24] [Q&A] [answered]
I am a starter learning to make hax like this, could u tell me how can i make a hot reload when modifying the code after injecting to game to apply changes. 
I try to eject the DLL and inject again, but usually, it cause game crash, and it waste more time.

**Comment by winstxnhdw** [comment, marked as answer]
Hot reloading depends on the injector. Currently, all versions of SharpMonoInjector, including my (superior) fork, does not unload the assembly safely. One day I might get around to fixing it, but for now, I have no idea why it does so.
